package.json

```diff
@@ -41,16 +41,15 @@
     "http.js",
     "https.js"
   ],
-  "dependencies": {
-    "debug": "^2.6.9"
-  },
+  "dependencies": {},
   "devDependencies": {
     "bluebird": "^3.4.0",
     "concat-stream": "^1.5.2",
-    "coveralls": "^2.11.15",
+    "coveralls": "^3.0.0",
+    "debug": "^3.1.0",
```

Comment:
This is not a `devDependency`; the package is required for the main software to work.